src/main/resources/application.yaml

```diff
@@ -17,4 +17,14 @@ jwt:
       key: encryptionSecret
 cors:
   hosts:
-    - http://localhost:4200
\ No newline at end of file
+    - http://localhost:4200
+cloud:
```

Comment:
this cloud specific information should not be in the application.yml
For local testing move it to the docker-compose, for cloud environment ask @el-dub to populate secrets on herocu